src/components/templates/newsAndEvents/newsAndEvents.scss

```
@@ -0,0 +1,48 @@
+@use "../../variables" as *;
+
+.newsEvents {
+  padding: 0rem 1rem;
+  background-color: $white_smoke;
+  .news-events {
+    padding: 56px 0px;
```

現状、`.container` に設定された横方向の padding を0 で上書きしてしまっています。
横幅が仕様より狭くなってしまっているので、修正しましょう。

```suggestion
    padding-top: 56px;
    padding-buttom: 56px;
```

こうすると横方向のpaddingは無指定になるので、 `.container` の方に書いてある内容が適用されます。
もしくは

```suggestion
    padding-block: 56px;
```

縦方向のpaddingを設定する [padding-block](https://developer.mozilla.org/ja/docs/Web/CSS/padding-block)を使ってもよいですね